### ISSUE TITLE
[generate:module] Module creation - adding dependencies throws undefined method

### ISSUE THREAD

**Issue by fluxsauce** [hechoendrupal/DrupalAppConsole#459]
````
Fatal error: Call to undefined method Drupal\AppConsole\EventSubscriber\ShowGeneratedFiles::trans() in ... ShowGeneratedFiles.php on line 60
````

**Comment by chipkaye** [comment]
Tested this on v0.7.2 and I am not seeing this error.  However, adding a dependency always displays the following warning - i.e. even when the module in question is enabled:

"Warning The following modules are not already available in your local environment "rest""

**Comment by jmolivas** [comment]
This was fixed with #453
